justfile

```diff
@@ -1,55 +1,100 @@
+# just has no idiom for setting an default value for an environment variable
+# so we shell out, as we need VIRTUAL_ENV in the justfile environment
+export VIRTUAL_ENV  := `echo ${VIRTUAL_ENV:-.venv}`
```

Comment:
Do the backticks make this shell out?

Reply:
yes, https://github.com/casey/just#command-evaluation-using-backticks

Reply:
yeah, makes sense, it took me a while to figure this out in just.

Make has a super conveinient syntax for this:

`NAME ?= value`

Tempted to see if I can add it to just, as it matchs the ':=` syntax they've got now